node/Makefile

```diff
@@ -252,9 +252,11 @@ endif
 # Building the image
 ###############################################################################
 ## Create the images for all supported ARCHes
-image-all: $(addprefix sub-image-,$(VALIDARCHES))
+image-all: $(addprefix sub-image-,$(VALIDARCHES)) sub-image-fips-amd64 image-windows
```

Comment:
I don't think we should include `image-windows` here, since its publish process is done separately

Reply:
OK. will remove. Thanks.